src/openrct2/paint/track/gentle/GhostTrain.cpp

```diff
@@ -602,7 +602,7 @@ TRACK_PAINT_FUNCTION GetTrackPaintFunctionGhostTrain(int32_t trackType)
 
         case TrackElemType::SpinningTunnel:
             return PaintGhostTrainTrackSpinningTunnel;
+        default:
+            return nullptr;
```

Comment:
We shouldn't use default for switches using a strong enum type, the compiler will warn if we are missing a case, when there is a default it will not do that.

Reply:
In case some enumerations have no drawing function its probably better to make it explicit by just putting all unsupported cases at the end and let them fallthrough to return null.

Reply:
There are 340 track element types, 99% of switches only check a small subset. Are you seriously suggesting adding more than 300 lines to every single one of them?

Reply:
I didn't realize there are that many, that definitely changes things a bit. This however doesn't change the fact that this makes things a bit error prone, I guess this will be less of a problem when we move this to be objects in where we can validate that on import. So nothing needs to be changed.